### DISCUSSION TITLE
Add Tidal as a music service link option

### DISCUSSION THREAD

**Discussion by autemhelm** [aleksey-saenko/MusicRecognizer#24] [Ideas]
App is outstanding, works just as well as Shazam, and it's FOSS! I'm sold! 

I use Tidal for my streaming service, and I didn't see it listed in the settings. Considering Tidal is one of the most popular services, would you consider adding it as one of the service link/button options?

Have a great day!

**Comment by aleksey-saenko** [comment]
Hey, thank you for your kind words!
Shazam has a couple of cool features that we don't have yet, but who knows, maybe someday they will be implemented :)
Also, it would be a truly free application if it used a free remote service as well. But I haven't found anything like that. There is [acoustid](https://acoustid.org/). It works well for unidentified studio recordings, but not for recordings from a microphone.
Regarding Tidal and other services - yes, there are some ideas on how to add them. You can read more about here #18 .

**Comment by aleksey-saenko** [comment]
@autemhelm support for Tidal service was added in v1.2.0. This feature only works for new recognitions and may take a few extra seconds after recognition to request Odesli service (sometimes, the link may not be found).
